src/Response/Exception/InvalidZipResponseException.php

```diff
@@ -0,0 +1,13 @@
+<?php
+declare(strict_types=1);
+
+namespace MarcusJaschen\Collmex\Response\Exception;
+
+/**
+ * This exception is thrown if an invalid Zip response is
```

Comment:
either:
"Zip response" -> "ZipResponse" (then it would be the class name)
of
"Zip response" -> "ZIP response"

Reply:
Update incoming…